content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -250,6 +250,7 @@ For a reference to old feature gates that are removed, please refer to
 | `DefaultPodTopologySpread` | `false` | Alpha | 1.19 | 1.19 |
 | `DefaultPodTopologySpread` | `true` | Beta | 1.20 | 1.23 |
 | `DefaultPodTopologySpread` | `true` | GA | 1.24 | - |
+| `DelegateFSGroupToCSIDriver` | `true` | GA | 1.26 |-|
```

Comment:
No. You are supposed to move all lines related to this feature to the second table. Check how we document other features?

Reply:
Moved the missing lines.